app/test/frontend/golden/v2/pkg_show_page_flutter_plugin.html

```diff
@@ -59,9 +59,8 @@ <h1 class="_visuallyhidden">Dart pub</h1>
 
   <div class="_banner-bg">
     <main class="package-banner">
-      <form class="search-bar" action="/experimental/search">
+      <form class="search-bar" action="/experimental/packages">
```

Comment:
/experimental/flutter/packages?

Reply:
This is a package show page, and we don't have good platform to chose (especially if it is not flutter-only).